dts/arm/st/h7/stm32h745.dtsi

```diff
@@ -21,11 +21,23 @@
 
 	/* System data RAM accessible over AHB bus: SRAM1 in D2 domain */
 	sram1: memory@30000000 {
-		reg = <0x30000000 DT_SIZE_K(288)>;
```

Comment:
This covered the whole D2 RAM: SRAM1, SRAM2 and SRAM3

I hope it is fine to separate it.

Reply:
I think, it is okay for the first time. Most important thing here is that, to know, how to generate linker sections with MPU configuration. Therefore it is not important, which and how much sram were used. Let it as it stays here. In the next step, 9t will be improved.